frontend/src/components/Navigator/ActivityReportNavigator.js

```diff
@@ -137,8 +138,15 @@ const ActivityReportNavigator = ({
     setError,
     watch,
     errors,
+    reset,
   } = hookForm;
 
+  // A new form page is being shown so we need to reset `react-hook-form` so validations are
+  // reset and the proper values are placed inside inputs
+  useDeepCompareEffect(() => {
```

Comment:
Change seems good as I no longer see the alert message. I'm wondering how safe of a change this is as it resets all form data.

Reply:
@AdamAdHocTeam This code that's in production right now (in fact, it's been part of the AR form since the beginning), this PR only puts it back.